### PR TITLE
Add line_profiler

### PR DESCRIPTION
**Pull request by jakirkham**
Adds the `line_profiler` package, which is great tool for optimizing code line-by-line and searching for inefficiencies.

### CODE REVIEW

**Comment by jakirkham** [comment]
Would you like me to add you, @koverholt?

**Comment by koverholt** [comment]
Was just expressing my ❤️  for this package, but I'm happy to be added as a maintainer if you'd like.

**Comment by jakirkham** [comment]
It's a great package for sure. Not trying to twist you're arm, but if you're interested I would be happy to have you on board.

**Comment by koverholt** [comment]
Sure thing, feel free to add me.

File: recipes/line_profiler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+about:
+  home: https://github.com/rkern/line_profiler
+  license: BSD 3-Clause
+  summary: A module for monitoring memory usage of a python program
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - koverholt
```
Comment (jakirkham) [inline recipes/line_profiler/meta.yaml]: Added. 😄

**Comment by jakirkham** [comment]
@rkern, just letting you know that we have added `line_profiler` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge line_profiler`]( http://anaconda.org/conda-forge/line_profiler ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/line_profiler-feedstock ). If you have any interest in being added as a maintainer (useful for making new releases for instance), we would be more than happy to add you to the maintainers list.

**Pull request merged**